website/docs/r/ssm_document.html.markdown

```diff
@@ -48,12 +48,21 @@ DOC
 The following arguments are supported:
 
 * `name` - (Required) The name of the document.
+* `attachments` - (Optional) A list of key/value pairs describing attachments to a version of a document. Defined below.
```

Comment:
Nit: Recommend shying away from "list" and "key/value pairs" when referring to a configuration block attributes, since in Terraform 0.12+ they all mean semantically different things (and Terraform 0.12 removed the old `attachments_source = [{}]` syntax you could do in some undocumented cases).

```suggestion
* `attachments_source` - (Optional) One or more configuration blocks describing attachments sources to a version of a document. Defined below.
```